CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 (Please put changes here)
 
+- Upgrade to botocore from `1.12.163` to `1.12.230`
+- The types `rusoto_events::{CloudWatchEvents,CloudWatchEventsClient}` were renamed to `rusoto_events::{EventBridge,EventBridgeClient}`
```

Comment:
@matthewkmayer its notable that the endpoint is still "events" despite the service being renamed. I just wanted to call that out [here](https://github.com/boto/botocore/blob/1.12.230/botocore/data/events/2015-10-07/service-2.json#L5)